internal/config/config.go

```diff
@@ -43,6 +43,7 @@ type Config struct {
 	CredentialsFile  string                 `yaml:"credentialsFile"`
 	Clusters         []ConfigCluster        `yaml:"clusters"`
 	Policies         []ConfigPolicy         `yaml:"policies"`
+	Inputs           []ConfigInput          `yaml:"inputs"`
```

Comment:
shouldn't we add also some documentation on that? how to configure Inputs in config yaml file?

Reply:
ok, I see - we can add it when we will switch to this approach